iOS/Flows/Home/_New/ConversationCollectionViewLayout.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  new_ConversationCollectionViewLayout.swift
+//  new_ConversationCollectionViewLayout
+//
+//  Created by Martin Young on 9/15/21.
+//  Copyright © 2021 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+
+class ConversationCollectionViewLayout: UICollectionViewCompositionalLayout {
```

Comment:
OK, here's the new stuff. Just the beginnings of the conversation layout. Pretty ugly for now but we'll come back to this.